Sources/_Datadog_TestRunner/DDTestRunner.swift

```diff
@@ -22,22 +23,15 @@ internal class DDTestRunner: NSObject, XCTestObservation {
     }
 
     func startTracer() {
-        var clientToken = ""
-        if let envToken = ProcessInfo.processInfo.environment["DATADOG_CLIENT_TOKEN"] {
-            clientToken = envToken.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
-        } else if let bundleToken = Bundle.main.infoDictionary?["DatadogClientToken"] as? String {
-            clientToken = bundleToken.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
-        }
-
-        if clientToken.isEmpty {
+        guard let clientToken = env.ddClientToken else {
             return
         }
 
         Datadog.initialize(
             appContext: .init(),
             configuration: Datadog.Configuration
-                .builderUsing(clientToken: clientToken, environment: CIEnvironmentValues.getEnvVariable("DD_ENV") ?? "test")
-                .set(serviceName: CIEnvironmentValues.getEnvVariable("DD_SERVICE") ?? ProcessInfo.processInfo.processName)
+                .builderUsing(clientToken: clientToken, environment: env.ddEnvironment ?? "test")
+                .set(serviceName: env.ddService ?? ProcessInfo.processInfo.processName)
```

Comment:
If not provided, `serviceName` defaults to the app bundle identifier, then to `"ios"`:
https://github.com/DataDog/dd-sdk-ios/blob/787a9d2a5d754a71e920f82839398135e2c229d3/Sources/Datadog/Core/FeaturesConfiguration.swift#L82
Not sure if we're interested in having the same defaults in the tests runner - but why not (all data is already segmented by the span type).